package.json

```diff
@@ -19,7 +19,7 @@
     "cbuild": "cbuild",
     "dump-lib": "dump-em-lib dist/em-api.js",
     "lint": "tslint -c src/tslint.json src/*.ts src/em/*.ts",
-    "prepublish": "npm run lint && tsc -p src/em && tsc -p src && cbuild -x -v -s dist/bundle/em/em-api.js -o dist/em-api.js",
+    "prepublish": "tsc -p src/em && tsc -p src && cbuild -x -v -s dist/bundle/em/em-api.js -o dist/em-api.js && npm run lint",
```

Comment:
This change was made to allow compiling the project even if lint eventually fails?